mender.go

```diff
@@ -503,14 +525,14 @@ func (m *mender) CheckUpdate() (*client.UpdateResponse, menderError) {
 
 func (m *mender) ReportUpdateStatus(update client.UpdateResponse, status string) menderError {
 	s := client.NewStatus()
-	err := s.Report(m.api.Request(m.authToken, reauthorize(m)), m.config.ServerURL,
+	err := s.Report(m.api.Request(m.authToken, setNextServer(m), reauthorize(m)), m.config.ServerURL,
 		client.StatusReport{
 			DeploymentID: update.ID,
 			Status:       status,
 		})
 	if err != nil {
 		log.Error("error reporting update status: ", err)
-
+		// lock your machine!
```

Comment:
@oleorhagen: Is this you? :-)

Reply:
haha, oops...

Reply:
haha, I believe not. @alfrunes has siblings, I think